app/Http/Resources/UserShortResource.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Interns2025b\Http\Resources;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class UserShortResource extends JsonResource
+{
+    public function toArray(Request $request): array
+    {
+        return [
+            "id" => $this->id,
+            "name" => $this->fullName,
```

Comment:
Shouldn't that be `full_name`?
We should add dock block in User model for this field

```suggestion
            "name" => $this->full_aame,
```